packages/components/data-table-manager/package.json

```diff
@@ -49,6 +49,8 @@
     "react-required-if": "1.0.3"
```

Comment:
This dependency can be removed now.

Reply:
Removed in this commit: 35ac3cf400bd274b73b64acbefb86aa0cf1efc17